nni/retiarii/codegen/pytorch.py

```diff
@@ -63,42 +67,37 @@ def _format_inputs(node: Node) -> Tuple[List[str], List[Any]]:
             assert isinstance(edge.head_slot, int)
             if edge.head.operation.io_names is not None:
                 # when input has names, e.g., forward(self, tensor1, tensor2, another_one)
-                inputs.append(edge.head.operation.io_names[edge.head_slot])
+                inputs.append(_format_variable_name(edge.head.operation.io_names[edge.head_slot], graph_name))
             else:
                 # when input has no name, e.g., forward(*_inputs)
                 inputs.append('_inputs[{}]'.format(edge.head_slot))
             inputs_value.append(None)
         else:
             if edge.head_slot is None:
                 # when the input comes from a single-output operator
-                inputs.append('{}'.format(edge.head.name))
+                inputs.append(_format_variable_name(edge.head.name, graph_name))
                 if edge.head.operation.type in ('prim::Constant', 'prim::GetAttr') and \
                         'value' in edge.head.operation.parameters:
                     inputs_value.append(edge.head.operation.parameters['value'])
                 else:
                     inputs_value.append(None)
             else:
                 # when the input comes from a multi-output operator: needs to know which one it comes from
-                inputs.append('{}[{}]'.format(edge.head.name, edge.head_slot))
+                inputs.append('{}[{}]'.format(_format_variable_name(edge.head.name, graph_name), edge.head_slot))
                 inputs_value.append(None)
     return inputs, inputs_value
 
 
-def _remove_prefix(names, graph_name):
+def _format_variable_name(name: str, graph_name: str) -> str:
     """
-    variables name (full name space) is too long,
-    shorten the name by removing the prefix ```graph_name```
+    1. replace invalid characters in node name
+    2. variables name (full name space) is too long, shorten the name by removing the prefix ```graph_name```
     """
-    if isinstance(names, list):
-        converted_names = []
-        for name in names:
-            if name.startswith(graph_name):
-                converted_names.append(name[len(graph_name):])
-            else:
-                converted_names.append(name)
-        return converted_names
-    else:
-        return names[len(graph_name):] if names.startswith(graph_name) else names
+    name = name[len(graph_name):] if name.startswith(graph_name) else name
+    name = name.replace('/', '__')
```

Comment:
why adding this line, seems there is no such logic in the previous version

Reply:
Previously, nodes in cell are named as `cell0__op1` because `/` is not supported in graph converter.
I changed them to `cell0/op1` and added support for `/` in converter here.

Reply:
good choice, but where did you do this? not in this pr?

Reply:
![image](https://user-images.githubusercontent.com/8463288/152909337-5db8164c-e888-43b4-ac01-15a28c1db518.png)

![image](https://user-images.githubusercontent.com/8463288/152909376-569eb565-41c7-4a61-b308-6a1e1a96a604.png)